cli/lsp/language_server.rs

```diff
@@ -2859,6 +2904,7 @@ impl Inner {
         ca_stores: None,
         ca_file: None,
         unsafely_ignore_certificate_errors: None,
+        unstable: true,
```

Comment:
Does this mean that setting `unstable` in ` .vscode/settings.json` has no effect anymore?

Reply:
No, this is only for caching with deno graph.